flask_login/__init__.py

```diff
@@ -28,15 +24,41 @@
                     encode_cookie, decode_cookie, make_next_param)
 
 
-# prevent pyflakes from failing from unused imports
-assert (COOKIE_NAME, COOKIE_DURATION, COOKIE_SECURE, COOKIE_HTTPONLY,
-        LOGIN_MESSAGE, LOGIN_MESSAGE_CATEGORY, REFRESH_MESSAGE,
-        REFRESH_MESSAGE_CATEGORY, ID_ATTRIBUTE, AUTH_HEADER_NAME)
-assert (LoginManager, UserMixin, AnonymousUserMixin)
-assert (user_logged_in, user_logged_out, user_loaded_from_cookie,
-        user_loaded_from_header, user_loaded_from_request,
-        user_login_confirmed, user_unauthorized, user_needs_refresh,
-        user_accessed, session_protected)
-assert (current_user, login_url, make_secure_token, login_fresh, login_user,
-        logout_user, confirm_login, login_required, fresh_login_required,
-        set_login_view, encode_cookie, decode_cookie, make_next_param)
+__all__ = [
```

Comment:
I couldn't get the tests to pass with the `assert` - not sure if due to a newer version of pyflakes, newer version of python, or some other reason.. at any rate, I get this in python 3.4 - not sure about earlier versions:

```python
In [1]: assert((False,))
<ipython-input-1-b82c5342027e>:1: SyntaxWarning: assertion is always true, perhaps remove parentheses?
```